lib/fibered_mysql2/fibered_database_connection_pool.rb

```diff
@@ -90,8 +156,6 @@ def new_cond
       FiberedConditionVariable.new(self)
     end
 
-    private
```

Comment:
This was needed to satisfy some unit tests that called private methods (and produced NoMethodErrors). I imagine it might be better to modify the tests?

Reply:
That's interesting that the tests passed before but don't anymore because of this. It might speak to something we don't have full control over, but I wouldn't be able to know for sure without digging in. Overall I don't think it's much of a problem making `mon_initialize` public instead of private.